### PR TITLE
CT-1460 Add sentry

### PR DESCRIPTION
**Pull request by flebel**
This PR introduces Sentry to report exceptions raised by the downloads/uploader CLI.

### CODE REVIEW

File: bin/conductor [hunk 1]
```diff
@@ -469,4 +495,16 @@ def run_downloader(args):
 
 if __name__ == '__main__':
     args = parse_args()
+
+    if conductor.CONFIG["error_reporting"]:
+        import logging
+        logging.getLogger('conductor').info("Automatic error reporting to Conductor is enabled. Disable by setting the --error_reporting argument to 'false'.")
+
+        sentry_sdk.init(dsn="https://76bc995e897d4244860276e2f99bc967@o306892.ingest.sentry.io/5302663",
```
Comment (hoolymama) [inline bin/conductor:503]: nit - maybe hold the URL in a const variable.

**Comment by jesseconductor** [comment]
@flebel Should we update the installers to indicate we have this feature? Add the option for users to turn it on/off?

**Comment by jesseconductor** [comment]
@flebel Sentry indicates it's possible to provide user feedback as well. Does this exist for desktop apps? IMO, we should prompt the user before sending anything. It's also not too difficult to write a small UI ourselves if we have to do this.

image

File: bin/conductor [hunk 2]
```diff
@@ -241,6 +243,14 @@ def parse_args():
                                      "everyday, while storing the last 7 days "
                                      "of logs"))
 
+    submit_parser.add_argument("--error_reporting",
+                               choices=[True, False],
+                               default=conductor.CONFIG.get("error_reporting"),
+                               type=cast_to_bool,
+                               help="Provide control over automatic error reporting to Conductor. "
+                                    "Automatic error reporting helps our engineers troubleshoot "
```
Comment (jesseconductor) [inline bin/conductor:251]: Missing an 'and' in the help parameter?

**Comment by flebel** [comment]
Sentry will only be enabled when running `conductor downloader` or `conductor uploader`, so that we don't get stack traces from people importing some of the client tools code. What would the integration with the installers look like?

We can't take custom user feedback was this will run from the cli, possibly on a headless system.

**Pull request merged**